services/aws-v4/src/provide_credential/assume_role.rs

```diff
@@ -273,7 +273,7 @@ mod tests {
             "wJalrXUtnFEMI/K7MDENG/bPxRfiCYzEXAMPLEKEY"
         );
         assert_eq!(
-            &resp.result.credentials.session_token,
+            resp.result.credentials.session_token.trim(),
```

Comment:
This is related to https://www.reddit.com/r/rust/comments/1me64g1/fyi_quickxml_038_no_longer_trims_whitespace_when/.

Not sure if it affects our use case. I hope it would not.